app/services/quote_service.rb

```diff
@@ -0,0 +1,14 @@
+require 'faraday'
+require 'json'
+
+class QuoteService
+
+ def self.conn
```

Comment:
Nice job breaking this out into its own method. Typically it will go under the keyword `private`. So it would look like:
```
private

def self.conn 
   Faraday.new(url: 'https://api.quotable.io')
end
```
Note that I left the `/` off of that url. That will actually go on our `conn.get` argument.